### ISSUE TITLE
R version control error for pophelper

### ISSUE THREAD

**Issue by aramo685** [royfrancis/pophelper#67]
Hi 
I am having some problem installing pophelper package in R version 3.6.0 for Windows 10. I am using the following code:

# 
install.packages(c("Cairo","ggplot2","gridExtra","gtable","tidyr","devtools"),dependencies=T)
install.packages('pophelper')
devtools::install_github('royfrancis/pophelper')
library(pophelper)

## 
WARNING: Rtools is required to build R packages but is not currently installed. Please download and install the appropriate version of Rtools before proceeding:

https://cran.rstudio.com/bin/windows/Rtools/
Installing package into ‘C:/Users/mgara/Documents/R/win-library/3.6’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘pophelper’ is not available (for R version 3.6.0)
> devtools::install_github('royfrancis/pophelper')
WARNING: Rtools is required to build R packages, but no version of Rtools compatible with R 3.6.0 was found. (Only the following incompatible version(s) of Rtools were found:4.0)

Please download and install Rtools 3.5 from https://cran.r-project.org/bin/windows/Rtools/.
Downloading GitHub repo royfrancis/pophelper@HEAD
WARNING: Rtools is required to build R packages, but no version of Rtools compatible with R 3.6.0 was found. (Only the following incompatible version(s) of Rtools were found:4.0)

Please download and install Rtools 3.5 from https://cran.r-project.org/bin/windows/Rtools/.
√  checking for file 'C:\Users\mgara\AppData\Local\Temp\RtmpMfDwoe\remotes1a0062772928\royfrancis-pophelper-3ad0dba/DESCRIPTION' ...
-  preparing 'pophelper':
√  checking DESCRIPTION meta-information ... 
-  checking for LF line-endings in source and make files and shell scripts
-  checking for empty or unneeded directories
   Removed empty directory 'pophelper/.github'
-  building 'pophelper_2.3.0.tar.gz'
   
Installing package into ‘C:/Users/mgara/Documents/R/win-library/3.6’
(as ‘lib’ is unspecified)
* installing *source* package 'pophelper' ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: (converted from warning) package 'ggplot2' was built under R version 3.6.3
Execution halted
ERROR: lazy loading failed for package 'pophelper'
* removing 'C:/Users/mgara/Documents/R/win-library/3.6/pophelper'
Error: Failed to install 'pophelper' from GitHub:
  (converted from warning) installation of package ‘C:/Users/mgara/AppData/Local/Temp/RtmpMfDwoe/file1a00846b23/pophelper_2.3.0.tar.gz’ had non-zero exit status

Please help me.
Regards
Motia

**Comment by royfrancis** [comment]
Yep. You still to install Rtools on windows to be able to build packages. Go [here](https://cran.r-project.org/bin/windows/Rtools/history.html) and download the Rtools version depending on your R version. Install it, restart your computer, then try the installation again.

**Comment by aramo685** [comment]
Hi again
Yes, I installed Rtools on windows and then restarted my computer and then tried the installation but again the same error message that- 
Installing package into ‘C:/Users/mgara/Documents/R/win-library/4.0’
(as ‘lib’ is unspecified)
* installing *source* package 'pophelper' ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: (converted from warning) package 'ggplot2' was built under R version 4.0.3
Execution halted
ERROR: lazy loading failed for package 'pophelper'
* removing 'C:/Users/mgara/Documents/R/win-library/4.0/pophelper'
Error: Failed to install 'pophelper' from GitHub:
  (converted from warning) installation of package ‘C:/Users/mgara/AppData/Local/Temp/Rtmp8OcIcv/file2d6417913c3e/pophelper_2.3.0.tar.gz’ had non-zero exit status

## what is mean by that - pophelper_2.3.0.tar.gz’ had non-zero exit status

Regards
Motia

**Comment by jks383** [comment]
I had the same issue, was this resolved?
